obs-packaging/gen_versions_txt.sh

```diff
@@ -40,8 +40,13 @@ gen_version_file() {
 	qemu_lite_hash=$(git ls-remote https://github.com/${project}/qemu.git | grep "refs/heads/${qemu_lite_branch}" | awk '{print $1}')
 
 	qemu_vanilla_branch=$(get_from_kata_deps "assets.hypervisor.qemu.version" "${kata_version}")
+	qemu_vanilla_ref="refs/heads/${qemu_vanilla_branch}"
+	if ! (git ls-remote --heads "https://github.com/qemu/qemu.git" | grep -q "refs/heads/${qemu_vanilla_branch}"); then
```

Comment:
May be worth adding a comment about qemu.ver / qemu.tag selection.

Reply:
done, thanks